ulb.py

```diff
@@ -197,40 +198,56 @@ def print_event(cpu, data, size):
     event = ct.cast(data, ct.POINTER(Data)).contents
     print("source {} --> dest {} {}".format(ip_mac_tostr(event.smac, event.saddr),ip_mac_tostr(event.dmac, event.daddr), associationType_tostr(event.associationType)))
 
+# Handle Signal
+class Stopper:
+  _stop = False
+  def __init__(self):
+    signal.signal(signal.SIGTERM, self.stop)
+    signal.signal(signal.SIGINT, self.stop) # keyboard interruption
+
+  def stop(self,signum, frame):
+    print ("\nStopping by signal {}({})...".format(signal.Signals(signum).name, signum));  
```

Comment:
s/Stopping by/Received/

Reply:
I want a message which show we are in a process about stopping the application.
So I prefer Stopping